modules/rostests/kmtests/ntos_se/SeQueryInfoToken.c

```diff
@@ -240,14 +239,14 @@ START_TEST(SeQueryInfoToken)
     //----------------------------------------------------------------//
 
     AccessState = ExAllocatePool(PagedPool, sizeof(ACCESS_STATE));
-    PsProcessType = ExAllocatePool(PagedPool, sizeof(OBJECT_TYPE));
-    AuxData = ExAllocatePool(PagedPool, 0xC8);
-    GenericMapping = ExAllocatePool(PagedPool, sizeof(GENERIC_MAPPING));
+    // AUX_ACCESS_DATA gets larger in newer Windows version.
+    // This is the largest known size, found in Windows 10/11.
+    AuxData = ExAllocatePool(PagedPool, 0xe0);
 
     Status = SeCreateAccessState(AccessState,
-                                 (PVOID)AuxData,
+                                 AuxData,
                                  DesiredAccess,
-                                 GenericMapping
+                                 &(*PsProcessType)->TypeInfo.GenericMapping
```

Comment:
This will fail/crash on Windows Vista SP1 and later (see http://terminus.rewolf.pl/terminus/structures/ntdll/_OBJECT_TYPE_x86.html). Why not just put a copy of the GENERIC_MAPPING into this file and use it here?

Reply:
Shouldn't also the kernel tests be rebuilt with suitable `NTDDI_VERSION` values and be run only on the windows version for which they have been compiled?

Reply:
The original design for kmtests had them running on Windows Server 2003 SP1 only and nothing else. But it's been shown that running them on other versions can be useful. There are a bunch of tests that make assumptions about internal data structures... but there's no reason this has to be one of them. So yeah, I'll go with Timo's fix.